tests/SimplifyTest.elm

```diff
@@ -10424,4 +10424,27 @@ a = (let b = c in (f x).e).f
 a = (let b = c in ((f x).e).f)
 """
                         ]
+        , test "should simplify record accesses for if/then/else expressions" <|
+            \() ->
+                """module A exposing (..)
+a = (if x then { f = 3 } else { z | f = 3 }).f
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "Field access can be simplified"
+                            , details = [ "Accessing the field outside an if/then/else expression can be simplified to access the field inside it" ]
+                            , under = "(if x then { f = 3 } else { z | f = 3 }).f"
```

Comment:
The under is getting really big here. In a real example, this would put squiggly lines over multiple lines (since ifs are always on multiple lines), which can be very distracting and annoying to a user.

I think that we should change the "under" to only be the `.f` in all of the simplifications we've done around record access. What do you think?

Reply:
Agreed!